File: CS/WindowsApplication3/MyObject.cs
using System;
using System.Collections.Generic;
using System.Windows.Forms;
using DevExpress.Skins;
using System.Drawing;
using System.ComponentModel;

namespace DXSample {
    public class MyObject {
        int id;
        string text;
        Point point;

        public MyObject(int id, string text, Point point) {
            this.id = id;
            this.text = text;
            this.point = point;
        }

        public int ID {
            get { return id; }
            set { id = value; }
        }

        public string Text {
            get { return text; }
            set { text = value; }
        }

        [TypeConverter(typeof(MyPointConverter))]
        public Point Point {
            get { return point; }
            set { point = value; }
        }
    }
}

File: VB/WindowsApplication3/MyObject.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Windows.Forms
Imports DevExpress.Skins
Imports System.Drawing
Imports System.ComponentModel

Namespace DXSample
	Public Class MyObject
		Private id_Renamed As Integer
		Private text_Renamed As String
		Private point_Renamed As Point

		Public Sub New(ByVal id As Integer, ByVal text As String, ByVal point As Point)
			Me.id_Renamed = id
			Me.text_Renamed = text
			Me.point_Renamed = point
		End Sub

		Public Property ID() As Integer
			Get
				Return id_Renamed
			End Get
			Set(ByVal value As Integer)
				id_Renamed = value
			End Set
		End Property

		Public Property Text() As String
			Get
				Return text_Renamed
			End Get
			Set(ByVal value As String)
				text_Renamed = value
			End Set
		End Property

		<TypeConverter(GetType(MyPointConverter))> _
		Public Property Point() As Point
			Get
				Return point_Renamed
			End Get
			Set(ByVal value As Point)
				point_Renamed = value
			End Set
		End Property
	End Class
End Namespace